src/main/windows/editor.js

```diff
@@ -47,11 +47,11 @@ class EditorWindow extends BaseWindow {
 
     // Enable native or custom/frameless window and titlebar
     const { titleBarStyle } = preferences.getAll()
-    if (titleBarStyle === 'custom') {
-      winOptions.titleBarStyle = ''
-    } else if (titleBarStyle === 'native') {
-      winOptions.frame = true
-      winOptions.titleBarStyle = ''
+    if (!isOsx) {
+      winOptions.titleBarStyle = 'default'
```

Comment:
`default` is not a valid value. I think `custom` could be the default one.

Reply:
>titleBarStyle String (optional) - The style of window title bar. Default is default.

According to Electron doc?

Reply:
Sorry, my bad.